compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -626,24 +628,30 @@ areNotFullTiles(ArrayRef<int64_t> inputShape,
   return false;
 }
 
+static SmallVector<OpFoldResult> getMixedValues(MLIRContext *context,
```

Comment:
Why the unnecessary indirection?

Reply:
I just didn't want to pass in the OpBuilder everywhere.